<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 2 Prerequisites | Introduction to pVACtools</title>
  <meta name="description" content="An introduction to the background and basic concepts around neoantigen identification and how to use pVACtools" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 2 Prerequisites | Introduction to pVACtools" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="An introduction to the background and basic concepts around neoantigen identification and how to use pVACtools" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 2 Prerequisites | Introduction to pVACtools" />
  
  <meta name="twitter:description" content="An introduction to the background and basic concepts around neoantigen identification and how to use pVACtools" />
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="assets/ITN_favicon.ico" type="image/x-icon" />
<link rel="prev" href="introduction.html"/>
<link rel="next" href="running-pvactools.html"/>
<script src="libs/header-attrs-2.10/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  background-color: #f8f8f8; }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #c4a000; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #000000; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #000000; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #000000; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="assets/style_ITN.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<a href="https://www.itcrtraining.org/"><img src="assets/ITN_logo.png" style="padding-left: 15px; padding-top: 8px;"</a>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>About this Course</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#motivation"><i class="fa fa-check"></i><b>1.1</b> Motivation</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#background"><i class="fa fa-check"></i><b>1.2</b> Background</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#target-audience"><i class="fa fa-check"></i><b>1.3</b> Target Audience</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#curriculum"><i class="fa fa-check"></i><b>1.4</b> Curriculum</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="prerequisites.html"><a href="prerequisites.html"><i class="fa fa-check"></i><b>2</b> Prerequisites</a>
<ul>
<li class="chapter" data-level="2.1" data-path="prerequisites.html"><a href="prerequisites.html#learning-objectives"><i class="fa fa-check"></i><b>2.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="2.2" data-path="prerequisites.html"><a href="prerequisites.html#docker"><i class="fa fa-check"></i><b>2.2</b> Docker</a></li>
<li class="chapter" data-level="2.3" data-path="prerequisites.html"><a href="prerequisites.html#terminal"><i class="fa fa-check"></i><b>2.3</b> Terminal</a></li>
<li class="chapter" data-level="2.4" data-path="prerequisites.html"><a href="prerequisites.html#r-studio-and-r-package-dependencies"><i class="fa fa-check"></i><b>2.4</b> R Studio and R package dependencies</a></li>
<li class="chapter" data-level="2.5" data-path="prerequisites.html"><a href="prerequisites.html#data"><i class="fa fa-check"></i><b>2.5</b> Data</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="running-pvactools.html"><a href="running-pvactools.html"><i class="fa fa-check"></i><b>3</b> Running pVACtools</a>
<ul>
<li class="chapter" data-level="3.1" data-path="running-pvactools.html"><a href="running-pvactools.html#learning-objectives-1"><i class="fa fa-check"></i><b>3.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="3.2" data-path="running-pvactools.html"><a href="running-pvactools.html#starting-docker"><i class="fa fa-check"></i><b>3.2</b> Starting Docker</a></li>
<li class="chapter" data-level="3.3" data-path="running-pvactools.html"><a href="running-pvactools.html#running-pvacseq"><i class="fa fa-check"></i><b>3.3</b> Running pVACseq</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="running-pvactools.html"><a href="running-pvactools.html#required-parameters-for-pvacseq"><i class="fa fa-check"></i><b>3.3.1</b> Required Parameters for pVACseq</a></li>
<li class="chapter" data-level="3.3.2" data-path="running-pvactools.html"><a href="running-pvactools.html#optional-parameters-for-pvacseq"><i class="fa fa-check"></i><b>3.3.2</b> Optional Parameters for pVACseq</a></li>
<li class="chapter" data-level="3.3.3" data-path="running-pvactools.html"><a href="running-pvactools.html#pvacseq-command"><i class="fa fa-check"></i><b>3.3.3</b> pVACseq Command</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="running-pvactools.html"><a href="running-pvactools.html#running-pvacfuse"><i class="fa fa-check"></i><b>3.4</b> Running pVACfuse</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="running-pvactools.html"><a href="running-pvactools.html#required-parameters-for-pvacfuse"><i class="fa fa-check"></i><b>3.4.1</b> Required Parameters for pVACfuse</a></li>
<li class="chapter" data-level="3.4.2" data-path="running-pvactools.html"><a href="running-pvactools.html#optional-parameters-for-pvacfuse"><i class="fa fa-check"></i><b>3.4.2</b> Optional Parameters for pVACfuse</a></li>
<li class="chapter" data-level="3.4.3" data-path="running-pvactools.html"><a href="running-pvactools.html#pvacfuse-command"><i class="fa fa-check"></i><b>3.4.3</b> pVACfuse Command</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html"><i class="fa fa-check"></i><b>4</b> Understanding pVACtools outputs</a>
<ul>
<li class="chapter" data-level="4.1" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#learning-objectives-2"><i class="fa fa-check"></i><b>4.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="4.2" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#pvactools-output-files"><i class="fa fa-check"></i><b>4.2</b> pVACtools Output Files</a></li>
<li class="chapter" data-level="4.3" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#interpreting-the-filtered.tsv-file"><i class="fa fa-check"></i><b>4.3</b> Interpreting the filtered.tsv File</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#binding-filter"><i class="fa fa-check"></i><b>4.3.1</b> Binding Filter</a></li>
<li class="chapter" data-level="4.3.2" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#coverage-filter"><i class="fa fa-check"></i><b>4.3.2</b> Coverage Filter</a></li>
<li class="chapter" data-level="4.3.3" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#transcript-support-level-filter"><i class="fa fa-check"></i><b>4.3.3</b> Transcript Support Level Filter</a></li>
<li class="chapter" data-level="4.3.4" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#top-score-filter"><i class="fa fa-check"></i><b>4.3.4</b> Top Score Filter</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#interpreting-the-aggregated.tsv-file"><i class="fa fa-check"></i><b>4.4</b> Interpreting the aggregated.tsv File</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#determining-the-best-transcript-and-best-peptide-of-a-variant"><i class="fa fa-check"></i><b>4.4.1</b> Determining the Best Transcript and Best Peptide of a Variant</a></li>
<li class="chapter" data-level="4.4.2" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#tier-and-tiering-criteria"><i class="fa fa-check"></i><b>4.4.2</b> Tier and Tiering Criteria</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html"><i class="fa fa-check"></i><b>5</b> Review Your Results in pVACview</a>
<ul>
<li class="chapter" data-level="5.1" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html#learning-objectives-3"><i class="fa fa-check"></i><b>5.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="5.2" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html#introduction-to-the-pvacview-module"><i class="fa fa-check"></i><b>5.2</b> Introduction to the pVACview module</a></li>
<li class="chapter" data-level="5.3" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html#starting-pvacview"><i class="fa fa-check"></i><b>5.3</b> Starting pVACview</a></li>
<li class="chapter" data-level="5.4" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html#uploading-pvacseq-result-data-into-pvacview"><i class="fa fa-check"></i><b>5.4</b> Uploading pVACseq result data into pVACview</a></li>
<li class="chapter" data-level="5.5" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html#the-pvacview-user-interface"><i class="fa fa-check"></i><b>5.5</b> The pVACview User Interface</a>
<ul>
<li class="chapter" data-level="5.5.1" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html#aggregate-report-of-best-candidates-by-variant"><i class="fa fa-check"></i><b>5.5.1</b> Aggregate Report of Best Candidates by Variant</a></li>
<li class="chapter" data-level="5.5.2" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html#variant-information"><i class="fa fa-check"></i><b>5.5.2</b> Variant Information</a></li>
<li class="chapter" data-level="5.5.3" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html#transcript-set-detailed-data"><i class="fa fa-check"></i><b>5.5.3</b> Transcript Set Detailed Data</a></li>
<li class="chapter" data-level="5.5.4" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html#additional-peptide-information"><i class="fa fa-check"></i><b>5.5.4</b> Additional Peptide Information</a></li>
</ul></li>
<li class="chapter" data-level="5.6" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html#regenerate-tiers-with-custom-parameters"><i class="fa fa-check"></i><b>5.6</b> Regenerate Tiers with Custom Parameters</a></li>
<li class="chapter" data-level="5.7" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html#adding-comments-to-variants"><i class="fa fa-check"></i><b>5.7</b> Adding Comments to Variants</a></li>
<li class="chapter" data-level="5.8" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html#exporting-the-aggregated-table"><i class="fa fa-check"></i><b>5.8</b> Exporting the Aggregated Table</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>6</b> Conclusions</a>
<ul>
<li class="chapter" data-level="6.1" data-path="conclusions.html"><a href="conclusions.html#learning-objectives-4"><i class="fa fa-check"></i><b>6.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="6.2" data-path="conclusions.html"><a href="conclusions.html#key-conclusions"><i class="fa fa-check"></i><b>6.2</b> Key conclusions</a></li>
<li class="chapter" data-level="6.3" data-path="conclusions.html"><a href="conclusions.html#additional-resources"><i class="fa fa-check"></i><b>6.3</b> Additional Resources</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="about-the-authors.html"><a href="about-the-authors.html"><i class="fa fa-check"></i>About the Authors</a></li>
<li class="chapter" data-level="7" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>7</b> References</a></li>
<li class="divider"></li>
<p style="text-align:center;"> <a href="https://github.com/jhudsl/OTTR_Template" target="blank" > This content was published with</a> <a href="https://bookdown.org/" target="blank"> bookdown by: </a> </p>
<p style="text-align:center;"> <a href="https://griffithlab.org/"> The Griffith Lab </a></p>
<p style="text-align:center; font-size: 12px;"> <a href="https://github.com/rstudio4edu/rstudio4edu-book/"> Style adapted from: rstudio4edu-book </a> <a href ="https://creativecommons.org/licenses/by/2.0/"> (CC-BY 2.0) </a></p>
<p style="padding-left: 40px;"><div class="trapezoid" style = "padding-left: 40px;"><span>  <a href="https://forms.gle/hc8Xt3Y2Znjb6M4Y7"> Click here to provide feedback</a> <img src="assets/itcr_arrow.png" style=" width: 10%" ></span></div></p>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Introduction to pVACtools</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<head>
  <meta name="viewport" content="width=device-width,minimum-scale=1.0,maximum-scale=1.0,initial-scale=1.0">
  <!--script src="https://kit.fontawesome.com/6a26f47516.js"></script-->
  <!--<script src="assets/hideOutput.js"></script>-->
  <link href="assets/style_ITN.css" rel="stylesheet">
</head>
        


<div class="hero-image-container"> 
  <img class= "hero-image" src="assets/itcr_main_image.png">
</div>
<div id="prerequisites" class="section level1" number="2">
<h1><span class="header-section-number">Chapter 2</span> Prerequisites</h1>
<div id="learning-objectives" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> Learning Objectives</h2>
<p>This chapter will cover the prerequisites for this course, including:</p>
<ul>
<li>Installing Docker</li>
<li>Installing R Studio</li>
<li>Downloading data files</li>
</ul>
</div>
<div id="docker" class="section level2" number="2.2">
<h2><span class="header-section-number">2.2</span> Docker</h2>
<p>For the purpose of this course, we will be using Docker to run pVACseq and
pVACfuse.
Docker is a tool that is used to automate the deployment of applications
in lightweight containers so that applications can work efficiently in
different environments in isolation. We provide versioned Docker containers
for all pVACtools <a href="https://github.com/griffithlab/pVACtools/releases">releases</a>
via <a href="https://hub.docker.com/r/griffithlab/pvactools">Docker Hub using the griffithlab/pvactools image name</a>.</p>
<p>In order to use Docker, you will to download the <a href="https://www.docker.com/get-started/">Docker Desktop software</a>.
Please ensure you select the correct install package for your operating
system.</p>
</div>
<div id="terminal" class="section level2" number="2.3">
<h2><span class="header-section-number">2.3</span> Terminal</h2>
<p>We will be running Docker from the command line on your preferred terminal
using the Docker command line interface (CLI). The Docker CLI is already
included with Docker Desktop. Most operating systems already
come with a Terminal application. If yours doesnâ€™t, you will need to first
install one.</p>
</div>
<div id="r-studio-and-r-package-dependencies" class="section level2" number="2.4">
<h2><span class="header-section-number">2.4</span> R Studio and R package dependencies</h2>
<p>In order to use pVACview, you will need to download R. Please refer
<a href="https://cran.rstudio.com/">here</a> for downloading R (version 3.5 and above
required). You may also take the additional step of <a href="https://www.rstudio.com/products/rstudio/download/">downloading R
studio</a> if
you are not familiar with launching R Shiny from the command line.</p>
<p>Additionally, there are a number of packages you will need to install in your R/R studio:</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="prerequisites.html#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="fu">install.packages</span>(<span class="st">&quot;shiny&quot;</span>, <span class="at">dependencies=</span><span class="cn">TRUE</span>)</span>
<span id="cb1-2"><a href="prerequisites.html#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="fu">install.packages</span>(<span class="st">&quot;ggplot2&quot;</span>, <span class="at">dependencies=</span><span class="cn">TRUE</span>)</span>
<span id="cb1-3"><a href="prerequisites.html#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="fu">install.packages</span>(<span class="st">&quot;DT&quot;</span>, <span class="at">dependencies=</span><span class="cn">TRUE</span>)</span>
<span id="cb1-4"><a href="prerequisites.html#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="fu">install.packages</span>(<span class="st">&quot;reshape2&quot;</span>, <span class="at">dependencies=</span><span class="cn">TRUE</span>)</span>
<span id="cb1-5"><a href="prerequisites.html#cb1-5" aria-hidden="true" tabindex="-1"></a><span class="fu">install.packages</span>(<span class="st">&quot;jsonlite&quot;</span>, <span class="at">dependencies=</span><span class="cn">TRUE</span>)</span>
<span id="cb1-6"><a href="prerequisites.html#cb1-6" aria-hidden="true" tabindex="-1"></a><span class="fu">install.packages</span>(<span class="st">&quot;tibble&quot;</span>, <span class="at">dependencies=</span><span class="cn">TRUE</span>)</span>
<span id="cb1-7"><a href="prerequisites.html#cb1-7" aria-hidden="true" tabindex="-1"></a><span class="fu">install.packages</span>(<span class="st">&quot;tidyr&quot;</span>, <span class="at">dependencies=</span><span class="cn">TRUE</span>)</span>
<span id="cb1-8"><a href="prerequisites.html#cb1-8" aria-hidden="true" tabindex="-1"></a><span class="fu">install.packages</span>(<span class="st">&quot;plyr&quot;</span>, <span class="at">dependencies=</span><span class="cn">TRUE</span>)</span>
<span id="cb1-9"><a href="prerequisites.html#cb1-9" aria-hidden="true" tabindex="-1"></a><span class="fu">install.packages</span>(<span class="st">&quot;dplyr&quot;</span>, <span class="at">dependencies=</span><span class="cn">TRUE</span>)</span>
<span id="cb1-10"><a href="prerequisites.html#cb1-10" aria-hidden="true" tabindex="-1"></a><span class="fu">install.packages</span>(<span class="st">&quot;shinydashboard&quot;</span>, <span class="at">dependencies=</span><span class="cn">TRUE</span>)</span>
<span id="cb1-11"><a href="prerequisites.html#cb1-11" aria-hidden="true" tabindex="-1"></a><span class="fu">install.packages</span>(<span class="st">&quot;shinydashboardPlus&quot;</span>, <span class="at">dependencies=</span><span class="cn">TRUE</span>)</span>
<span id="cb1-12"><a href="prerequisites.html#cb1-12" aria-hidden="true" tabindex="-1"></a><span class="fu">install.packages</span>(<span class="st">&quot;fresh&quot;</span>, <span class="at">dependencies=</span><span class="cn">TRUE</span>)</span>
<span id="cb1-13"><a href="prerequisites.html#cb1-13" aria-hidden="true" tabindex="-1"></a><span class="fu">install.packages</span>(<span class="st">&quot;shinycssloaders&quot;</span>, <span class="at">dependencies=</span><span class="cn">TRUE</span>)</span>
<span id="cb1-14"><a href="prerequisites.html#cb1-14" aria-hidden="true" tabindex="-1"></a><span class="fu">install.packages</span>(<span class="st">&quot;RCurl&quot;</span>, <span class="at">dependencies=</span><span class="cn">TRUE</span>)</span>
<span id="cb1-15"><a href="prerequisites.html#cb1-15" aria-hidden="true" tabindex="-1"></a><span class="fu">install.packages</span>(<span class="st">&quot;curl&quot;</span>, <span class="at">dependencies=</span><span class="cn">TRUE</span>)</span>
<span id="cb1-16"><a href="prerequisites.html#cb1-16" aria-hidden="true" tabindex="-1"></a><span class="fu">install.packages</span>(<span class="st">&quot;string&quot;</span>, <span class="at">dependencies=</span><span class="cn">TRUE</span>)</span>
<span id="cb1-17"><a href="prerequisites.html#cb1-17" aria-hidden="true" tabindex="-1"></a><span class="fu">install.packages</span>(<span class="st">&quot;shinycssloaders&quot;</span>, <span class="at">dependencies=</span><span class="cn">TRUE</span>)</span></code></pre></div>
</div>
<div id="data" class="section level2" number="2.5">
<h2><span class="header-section-number">2.5</span> Data</h2>
<p>For this course, we have put together a set of input data generated from the breast
cancer cell line HCC1395 and a matched normal lymphoblastoid cell line HCC1395BL.
Data from this cell line is commonly used as test data in bioinformatics applications.
For more information on these lines and the generation of test data, please refer to
the <a href="https://pmbio.org/module-02-inputs/0002/05/01/Data/">data section of our precision medicine bioinformatics course</a>.</p>
<p>The input data consists of the following files:</p>
<p>For pVACseq:</p>
<ul>
<li><code>annotated.expression.vcf.gz</code>: A somatic (tumor-normal) VCF and its tbi index file. The VCF has been
annotated with VEP and has coverage and expression information added. It has also been annotated with
custom VEP plugins that provide wild type and mutant versions of the full length protein sequences
predicted to arise from each transcript annotated with each variant.</li>
<li><code>phased.vcf.gz</code>: A phased tumor-germline VCF and its tbi index file to provide information about
in-phase proximal variants that might alter the predicted peptide sequence around a somatic
mutation of interest.</li>
<li><code>optitype_normal_result.tsv</code>: A OptiType file with HLA allele typing predictions.</li>
</ul>
<p>For more detailed information on how the variant input file is created, please refer to the
<a href="https://pvactools.readthedocs.io/en/latest/pvacseq/input_file_prep.html">input file preparation</a>
section of the pVACtools docs.</p>
<p>For pVACfuse:</p>
<ul>
<li><code>agfusion_results</code>: An AGFusion output directory with annotated fusion
calls.</li>
<li><code>star-fusion.fusion_predictions.tsv</code>: A STARFusion prediction file with fusion read support
and expression information.</li>
</ul>
<p>General:</p>
<ul>
<li><code>Homo_sapiens.GRCh38.pep.all.fa.gz</code>: A reference proteome peptide FASTA to use
for determining whether there are any reference matches of neoantigen candidates.</li>
</ul>
<p>To download this data, please run the following commands:</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode bash"><code class="sourceCode bash"><span id="cb2-1"><a href="prerequisites.html#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="fu">wget</span> https://raw.githubusercontent.com/griffithlab/pVACtools_Intro_Course/main/HCC1395_inputs.zip</span>
<span id="cb2-2"><a href="prerequisites.html#cb2-2" aria-hidden="true" tabindex="-1"></a><span class="fu">unzip</span> HCC1395_inputs.zip</span></code></pre></div>
<p>This course will not cover the required pre-processing steps for the pVACtools
input data but extensive instructions on how to prepare your own data for use
with pVACtools can be found at <a href="http://www.pvactools.org">pvactools.org</a>.</p>

</div>
</div>
<hr>
<center> 
  <div class="footer">
      All illustrations <a href="https://creativecommons.org/licenses/by/4.0/">CC-BY. </a>
      <br>
      All other materials <a href= "https://creativecommons.org/licenses/by/4.0/"> CC-BY </a> unless noted otherwise.
  </div>
</center>
            </section>

          </div>
        </div>
      </div>
<a href="introduction.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="running-pvactools.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
