<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 5 Review Your Results in pVACview | Introduction to pVACtools</title>
  <meta name="description" content="An introduction to the background and basic concepts around neoantigen identification and how to use pVACtools" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 5 Review Your Results in pVACview | Introduction to pVACtools" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="An introduction to the background and basic concepts around neoantigen identification and how to use pVACtools" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 5 Review Your Results in pVACview | Introduction to pVACtools" />
  
  <meta name="twitter:description" content="An introduction to the background and basic concepts around neoantigen identification and how to use pVACtools" />
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="assets/ITN_favicon.ico" type="image/x-icon" />
<link rel="prev" href="understanding-pvactools-outputs.html"/>
<link rel="next" href="conclusions.html"/>
<script src="libs/header-attrs-2.10/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  background-color: #f8f8f8; }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #c4a000; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #000000; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #000000; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #000000; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="assets/style_ITN.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<a href="https://www.itcrtraining.org/"><img src="assets/ITN_logo.png" style="padding-left: 15px; padding-top: 8px;"</a>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>About this Course</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#motivation"><i class="fa fa-check"></i><b>1.1</b> Motivation</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#background"><i class="fa fa-check"></i><b>1.2</b> Background</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#target-audience"><i class="fa fa-check"></i><b>1.3</b> Target Audience</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#curriculum"><i class="fa fa-check"></i><b>1.4</b> Curriculum</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="prerequisites.html"><a href="prerequisites.html"><i class="fa fa-check"></i><b>2</b> Prerequisites</a>
<ul>
<li class="chapter" data-level="2.1" data-path="prerequisites.html"><a href="prerequisites.html#learning-objectives"><i class="fa fa-check"></i><b>2.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="2.2" data-path="prerequisites.html"><a href="prerequisites.html#docker"><i class="fa fa-check"></i><b>2.2</b> Docker</a></li>
<li class="chapter" data-level="2.3" data-path="prerequisites.html"><a href="prerequisites.html#terminal"><i class="fa fa-check"></i><b>2.3</b> Terminal</a></li>
<li class="chapter" data-level="2.4" data-path="prerequisites.html"><a href="prerequisites.html#r-studio-and-r-package-dependencies"><i class="fa fa-check"></i><b>2.4</b> R Studio and R package dependencies</a></li>
<li class="chapter" data-level="2.5" data-path="prerequisites.html"><a href="prerequisites.html#data"><i class="fa fa-check"></i><b>2.5</b> Data</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="running-pvactools.html"><a href="running-pvactools.html"><i class="fa fa-check"></i><b>3</b> Running pVACtools</a>
<ul>
<li class="chapter" data-level="3.1" data-path="running-pvactools.html"><a href="running-pvactools.html#learning-objectives-1"><i class="fa fa-check"></i><b>3.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="3.2" data-path="running-pvactools.html"><a href="running-pvactools.html#starting-docker"><i class="fa fa-check"></i><b>3.2</b> Starting Docker</a></li>
<li class="chapter" data-level="3.3" data-path="running-pvactools.html"><a href="running-pvactools.html#running-pvacseq"><i class="fa fa-check"></i><b>3.3</b> Running pVACseq</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="running-pvactools.html"><a href="running-pvactools.html#required-parameters-for-pvacseq"><i class="fa fa-check"></i><b>3.3.1</b> Required Parameters for pVACseq</a></li>
<li class="chapter" data-level="3.3.2" data-path="running-pvactools.html"><a href="running-pvactools.html#optional-parameters-for-pvacseq"><i class="fa fa-check"></i><b>3.3.2</b> Optional Parameters for pVACseq</a></li>
<li class="chapter" data-level="3.3.3" data-path="running-pvactools.html"><a href="running-pvactools.html#pvacseq-command"><i class="fa fa-check"></i><b>3.3.3</b> pVACseq Command</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="running-pvactools.html"><a href="running-pvactools.html#running-pvacfuse"><i class="fa fa-check"></i><b>3.4</b> Running pVACfuse</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="running-pvactools.html"><a href="running-pvactools.html#required-parameters-for-pvacfuse"><i class="fa fa-check"></i><b>3.4.1</b> Required Parameters for pVACfuse</a></li>
<li class="chapter" data-level="3.4.2" data-path="running-pvactools.html"><a href="running-pvactools.html#optional-parameters-for-pvacfuse"><i class="fa fa-check"></i><b>3.4.2</b> Optional Parameters for pVACfuse</a></li>
<li class="chapter" data-level="3.4.3" data-path="running-pvactools.html"><a href="running-pvactools.html#pvacfuse-command"><i class="fa fa-check"></i><b>3.4.3</b> pVACfuse Command</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html"><i class="fa fa-check"></i><b>4</b> Understanding pVACtools outputs</a>
<ul>
<li class="chapter" data-level="4.1" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#learning-objectives-2"><i class="fa fa-check"></i><b>4.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="4.2" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#pvactools-output-files"><i class="fa fa-check"></i><b>4.2</b> pVACtools Output Files</a></li>
<li class="chapter" data-level="4.3" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#interpreting-the-filtered.tsv-file"><i class="fa fa-check"></i><b>4.3</b> Interpreting the filtered.tsv File</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#binding-filter"><i class="fa fa-check"></i><b>4.3.1</b> Binding Filter</a></li>
<li class="chapter" data-level="4.3.2" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#coverage-filter"><i class="fa fa-check"></i><b>4.3.2</b> Coverage Filter</a></li>
<li class="chapter" data-level="4.3.3" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#transcript-support-level-filter"><i class="fa fa-check"></i><b>4.3.3</b> Transcript Support Level Filter</a></li>
<li class="chapter" data-level="4.3.4" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#top-score-filter"><i class="fa fa-check"></i><b>4.3.4</b> Top Score Filter</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#interpreting-the-aggregated.tsv-file"><i class="fa fa-check"></i><b>4.4</b> Interpreting the aggregated.tsv File</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#determining-the-best-transcript-and-best-peptide-of-a-variant"><i class="fa fa-check"></i><b>4.4.1</b> Determining the Best Transcript and Best Peptide of a Variant</a></li>
<li class="chapter" data-level="4.4.2" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#tier-and-tiering-criteria"><i class="fa fa-check"></i><b>4.4.2</b> Tier and Tiering Criteria</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html"><i class="fa fa-check"></i><b>5</b> Review Your Results in pVACview</a>
<ul>
<li class="chapter" data-level="5.1" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html#learning-objectives-3"><i class="fa fa-check"></i><b>5.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="5.2" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html#introduction-to-the-pvacview-module"><i class="fa fa-check"></i><b>5.2</b> Introduction to the pVACview module</a></li>
<li class="chapter" data-level="5.3" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html#starting-pvacview"><i class="fa fa-check"></i><b>5.3</b> Starting pVACview</a></li>
<li class="chapter" data-level="5.4" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html#uploading-pvacseq-result-data-into-pvacview"><i class="fa fa-check"></i><b>5.4</b> Uploading pVACseq result data into pVACview</a></li>
<li class="chapter" data-level="5.5" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html#the-pvacview-user-interface"><i class="fa fa-check"></i><b>5.5</b> The pVACview User Interface</a>
<ul>
<li class="chapter" data-level="5.5.1" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html#aggregate-report-of-best-candidates-by-variant"><i class="fa fa-check"></i><b>5.5.1</b> Aggregate Report of Best Candidates by Variant</a></li>
<li class="chapter" data-level="5.5.2" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html#variant-information"><i class="fa fa-check"></i><b>5.5.2</b> Variant Information</a></li>
<li class="chapter" data-level="5.5.3" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html#transcript-set-detailed-data"><i class="fa fa-check"></i><b>5.5.3</b> Transcript Set Detailed Data</a></li>
<li class="chapter" data-level="5.5.4" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html#additional-peptide-information"><i class="fa fa-check"></i><b>5.5.4</b> Additional Peptide Information</a></li>
</ul></li>
<li class="chapter" data-level="5.6" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html#regenerate-tiers-with-custom-parameters"><i class="fa fa-check"></i><b>5.6</b> Regenerate Tiers with Custom Parameters</a></li>
<li class="chapter" data-level="5.7" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html#adding-comments-to-variants"><i class="fa fa-check"></i><b>5.7</b> Adding Comments to Variants</a></li>
<li class="chapter" data-level="5.8" data-path="review-your-results-in-pvacview.html"><a href="review-your-results-in-pvacview.html#exporting-the-aggregated-table"><i class="fa fa-check"></i><b>5.8</b> Exporting the Aggregated Table</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>6</b> Conclusions</a>
<ul>
<li class="chapter" data-level="6.1" data-path="conclusions.html"><a href="conclusions.html#learning-objectives-4"><i class="fa fa-check"></i><b>6.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="6.2" data-path="conclusions.html"><a href="conclusions.html#key-conclusions"><i class="fa fa-check"></i><b>6.2</b> Key conclusions</a></li>
<li class="chapter" data-level="6.3" data-path="conclusions.html"><a href="conclusions.html#additional-resources"><i class="fa fa-check"></i><b>6.3</b> Additional Resources</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="about-the-authors.html"><a href="about-the-authors.html"><i class="fa fa-check"></i>About the Authors</a></li>
<li class="chapter" data-level="7" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>7</b> References</a></li>
<li class="divider"></li>
<p style="text-align:center;"> <a href="https://github.com/jhudsl/OTTR_Template" target="blank" > This content was published with</a> <a href="https://bookdown.org/" target="blank"> bookdown by: </a> </p>
<p style="text-align:center;"> <a href="https://griffithlab.org/"> The Griffith Lab </a></p>
<p style="text-align:center; font-size: 12px;"> <a href="https://github.com/rstudio4edu/rstudio4edu-book/"> Style adapted from: rstudio4edu-book </a> <a href ="https://creativecommons.org/licenses/by/2.0/"> (CC-BY 2.0) </a></p>
<p style="padding-left: 40px;"><div class="trapezoid" style = "padding-left: 40px;"><span>  <a href="https://forms.gle/hc8Xt3Y2Znjb6M4Y7"> Click here to provide feedback</a> <img src="assets/itcr_arrow.png" style=" width: 10%" ></span></div></p>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Introduction to pVACtools</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<head>
  <meta name="viewport" content="width=device-width,minimum-scale=1.0,maximum-scale=1.0,initial-scale=1.0">
  <!--script src="https://kit.fontawesome.com/6a26f47516.js"></script-->
  <!--<script src="assets/hideOutput.js"></script>-->
  <link href="assets/style_ITN.css" rel="stylesheet">
</head>
        


<div class="hero-image-container"> 
  <img class= "hero-image" src="assets/itcr_main_image.png">
</div>
<div id="review-your-results-in-pvacview" class="section level1" number="5">
<h1><span class="header-section-number">Chapter 5</span> Review Your Results in pVACview</h1>
<div id="learning-objectives-3" class="section level2" number="5.1">
<h2><span class="header-section-number">5.1</span> Learning Objectives</h2>
<p>This chapter will cover:</p>
<ul>
<li>Introduction to the pVACview module</li>
<li>How to start pVACview</li>
<li>How to load your pVACseq data into pVACview</li>
<li>The pVACview user interface</li>
<li>How to re-tier results in pVACview</li>
<li>How to add comments to the variants in the report</li>
<li>How to export the reviewed results</li>
</ul>
</div>
<div id="introduction-to-the-pvacview-module" class="section level2" number="5.2">
<h2><span class="header-section-number">5.2</span> Introduction to the pVACview module</h2>
<p>pVACview is a R shiny based tool designed to aid specifically in the prioritization and selection of
neoantigen candidates for personalized cancer vaccines or other applications. It takes as inputs a
pVACseq output aggregate report file (tsv format) and a corresponding pVACseq output metrics file (json).
pVACview allows the user to launch an R shiny application to load and visualize the given neoantigen
candidates with detailed information including that of the genomic variant, transcripts covering the
variant, and strong-binding peptides predicted from the respective transcripts. It also incorporates
anchor prediction data for a range of class I HLA alleles and peptides ranging from 8- to 11-mers. By
taking all these types of information into account for the neoantigen candidates, researchers will be
able to make more informed decisions when deciding final peptide candidates for experiments, personalized
cancer vaccines, or T cell therapies designed to target neoantigens.</p>
</div>
<div id="starting-pvacview" class="section level2" number="5.3">
<h2><span class="header-section-number">5.3</span> Starting pVACview</h2>
<p>The pVACview R source code is distributed with every pVACseq run the in
MHC_Class_I and/or MHC_Class_II subdirectories, depending on which prediction
algorithms were run. For the HCC1395 pVACseq example run you did earlier, it
can be started by running the following command in your Terminal:</p>
<div class="sourceCode" id="cb7"><pre class="sourceCode bash"><code class="sourceCode bash"><span id="cb7-1"><a href="review-your-results-in-pvacview.html#cb7-1" aria-hidden="true" tabindex="-1"></a><span class="ex">pvacview</span> run <span class="va">${PWD}</span>/pVACtools_outputs/MHC_Class_I</span></code></pre></div>
<p>This starts a process for the pVACview R shiny application. Do not close the
terminal window or end this process. You can now open pVACview in your browser
by navigating to <a href="http://127.0.0.1:3333/">http://127.0.0.1:3333/</a>.</p>
</div>
<div id="uploading-pvacseq-result-data-into-pvacview" class="section level2" number="5.4">
<h2><span class="header-section-number">5.4</span> Uploading pVACseq result data into pVACview</h2>
<p>The starting screen shows an upload form. You will want to upload the Class I
aggregated.tsv report under “1. Neoantigen Candidate Aggregate Report (tsv
required)”. We generally recommend using Class I as the main report, as these
are more robust than Class II. However, either can be uploaded here as long as
the correct data type is selected under “Does this aggregate report file
correspond to Class I or Class II prediction data?”</p>
<p>Upload the matching Class I metrics.json file under “2. Neoantigen Candidate
Metrics file (json required)”. The Class II aggregated.tsv can now be uploaded
as a supplementary file under “3. Additional Neoantigen Candidate Aggregate
Report (tsv required)”.</p>
<p>This interface also optionally allows users to upload a gene-of-interest TSV
file to, e.g. highlight variants on known cancer genes. We have provided such
a TSV with your original HCC1395 downloaded data, called
“cancer_census_hotspot_gene_list.tsv”.</p>
<p>After uploading all files, you can visualize the result by clicking on the
“Visualize” button at the bottom of the page, or the “Visualize and Explore”
tab in the sidebar.</p>
<p><img src="resources/images/05-pvacview_tour_files/figure-html/1uz39zaObDGKhEVCGzO0JO35CTbC0oRAM0mxgLcMAA9Y_g25ad9ce8c9b_0_2.png" title="pVACview data upload interface." alt="pVACview data upload interface." width="100%" style="display: block; margin: auto;" /></p>
</div>
<div id="the-pvacview-user-interface" class="section level2" number="5.5">
<h2><span class="header-section-number">5.5</span> The pVACview User Interface</h2>
<div id="aggregate-report-of-best-candidates-by-variant" class="section level3" number="5.5.1">
<h3><span class="header-section-number">5.5.1</span> Aggregate Report of Best Candidates by Variant</h3>
<p>The main table in the Aggregate Report of Best Candidates by Variant panel
shows the best neoantigen candidate for each variant.
It lists the gene and amino acid change of the variant as well as additional
information about the best peptide and the best transcript coding for it. These
include, from left to right, the transcript support level, the best-binding HLA
allele, the mutated positions of the best peptide, any positions in the peptide
where the amino acid might be problematic for manufacturing, and the total number
of neoantigen candidates passing the binding affinity threshold set by the user.
If a gene of interest list was uploaded, variants on those genes have their gene
highlighted with a green border.</p>
<p>Next, this table lists the IC50 peptide MHC binding affinity for both the mutant
and the wild type. It also shows the percentile scores of the binding affinity values.
For the mutant values, a heatmap coloring is applied to make it easier to visually
identify well-binding peptides.</p>
<p>The next few columns show the coverage and expression of the best transcript with
a bar plot background to represent where specific values fall across the entire
patient sample.</p>
<p>The Tier column represents the tier assigned to the best peptide. The neoantigen
candidates in this view were all sorted into the Pass tier but tiers such as Low
Expression or Subclonal are applied to easily identify why a neoantigen candidate
might be unsuitable for vaccine selection.</p>
<p>The Ref Match column reflects whether or not the best peptide was found in the
reference proteome which is undesired since such peptides are not novel and
including them in a vaccine might lead to an auto immune response.</p>
<p>Users are able to set a status for each candidate in the Evaluation column to mark
them as Accept, Reject, or requiring further review.</p>
<p>The Investigate button can be clicked to see more detail for a variant. This
will update the lower panels with details for the selected variant.</p>
<p><img src="resources/images/05-pvacview_tour_files/figure-html/1uz39zaObDGKhEVCGzO0JO35CTbC0oRAM0mxgLcMAA9Y_g2491f283519_0_8.png" title="Upon successfully uploading the relevant data files, you can explore the different aspects of your neoantigen candidates." alt="Upon successfully uploading the relevant data files, you can explore the different aspects of your neoantigen candidates." width="100%" style="display: block; margin: auto;" /></p>
<p>For candidates not sorted into the Pass tier, red borders visually highlight the
attributes failed by the candidate.</p>
<p><img src="resources/images/05-pvacview_tour_files/figure-html/1uz39zaObDGKhEVCGzO0JO35CTbC0oRAM0mxgLcMAA9Y_g25ad9ce8c9b_0_8.png" title="Neoantigen candidates are binned into tiers depending on their suitability for vaccine creation." alt="Neoantigen candidates are binned into tiers depending on their suitability for vaccine creation." width="100%" style="display: block; margin: auto;" /></p>
</div>
<div id="variant-information" class="section level3" number="5.5.2">
<h3><span class="header-section-number">5.5.2</span> Variant Information</h3>
<p>The Variant Information panel shows more variant-level details of the selected
neoantigen candidate.</p>
<p><img src="resources/images/05-pvacview_tour_files/figure-html/1uz39zaObDGKhEVCGzO0JO35CTbC0oRAM0mxgLcMAA9Y_g25ad9ce8c9b_0_14.png" title="The Variant Information tab shows more details for the selected variant." alt="The Variant Information tab shows more details for the selected variant." width="100%" style="display: block; margin: auto;" /></p>
<p>On the left is a three-tab section. The first tab, “Transcript Sets of Selected
Variant”, shows a list of transcript sets. pVACtools bins transcripts that code for
the same set of neoantigen candidates into a set
because the neoantigen-level information for all transcripts in a set will be
identical. The transcript set containing the Best Peptide is highlighted in
green. This table shows the number of transcripts in each set, the number of
well-binding peptides the transcripts in the set code for, and the total
expression of all the transcripts in the set.</p>
<p>The second tab, titled Reference Matches, shows the details for any reference matches of the neoantigen
candidate. It repeats information about the Best Peptide sequence, the amino acid
change, the mutated position, and the gene for easy reference. The mutated
positions are notated in red in the Best Peptide sequence and the Query
Sequence. The Query Sequence is a longer sequence around the somatic mutation.
The Best Peptide subsequence is highlighted in yellow in the Query Sequence. For
any 8mer subsequence of the query sequence, pVACtools looks for matches in the
reference proteome. Any matches found are reported in the Hits table</p>
<p><img src="resources/images/05-pvacview_tour_files/figure-html/1uz39zaObDGKhEVCGzO0JO35CTbC0oRAM0mxgLcMAA9Y_g25ad9ce8c9b_0_20.png" title="The Reference Matches tab shows details of reference matches of the neoantigen candidate." alt="The Reference Matches tab shows details of reference matches of the neoantigen candidate." width="100%" style="display: block; margin: auto;" /></p>
<p>The “Additional Data” tab shows the additional data for the variant if a Additional Neoantigen Candidate Aggregate
Report was uploaded. It shows the Best Peptide and its information for this variant from the
additional report. This can be used, e.g., when a Class I neoantigen candidate
is a bad binder but all other metrics look good. Oftentimes this variant can
be rescued by considering the best Class II neoantigen candidate instead.</p>
<p><img src="resources/images/05-pvacview_tour_files/figure-html/1uz39zaObDGKhEVCGzO0JO35CTbC0oRAM0mxgLcMAA9Y_g25ad9ce8c9b_0_26.png" title="The Additional Data shows data from the Additional Aggregate Report for the variant." alt="The Additional Data shows data from the Additional Aggregate Report for the variant." width="100%" style="display: block; margin: auto;" /></p>
<p>The next section, “Variant &amp; Gene Info”, shows coverage and expression information as well as
the genomic coordinates for the variant.</p>
<p><img src="resources/images/05-pvacview_tour_files/figure-html/1uz39zaObDGKhEVCGzO0JO35CTbC0oRAM0mxgLcMAA9Y_g25ad9ce8c9b_0_75.png" title="The Variant &amp; Gene INfo section show coverage and expression information for the variant." alt="The Variant &amp; Gene INfo section show coverage and expression information for the variant." width="100%" style="display: block; margin: auto;" /></p>
<p>The last section, “Peptide Evaluation Overview”, shows counts for how many peptides have been accepted,
rejected, or marked for review. For most vaccines a certain minimum number of
neoantigen candidates is desired so this panel makes it easy to review how
many neoantigen candidates are still needed.</p>
<p><img src="resources/images/05-pvacview_tour_files/figure-html/1uz39zaObDGKhEVCGzO0JO35CTbC0oRAM0mxgLcMAA9Y_g25ad9ce8c9b_0_80.png" title="The Peptide Evaluation Overview section shows how many neoantigen candidates have been accepted, rejected, marked for review, or are pending." alt="The Peptide Evaluation Overview section shows how many neoantigen candidates have been accepted, rejected, marked for review, or are pending." width="100%" style="display: block; margin: auto;" /></p>
</div>
<div id="transcript-set-detailed-data" class="section level3" number="5.5.3">
<h3><span class="header-section-number">5.5.3</span> Transcript Set Detailed Data</h3>
<p>When selecting a transcript set in the Variant Info panel, this panel will
show details about the neoantigen candidates the transcripts in the set code
for and as well as details on the transcripts themselves.</p>
<p>The Peptide Candidates from Selected Transcript Set tab shows a list of
mutant and matched wildtype peptides and their IC50 binding
affinity to the patient HLA alleles. Only neoantigen candidates where at least
one peptide-MHC binding prediction falls within the <code>--aggregate-inclusion-threshold</code>
will be shown in this table. For HLA alleles where the peptide is not
well-binding the prediction details will show <code>X</code>. This table also shows the
mutant position, whether or not the neoantigen candidate has any problematic
positions, and whether or not it failed the anchor criteria. This helps in
determining whether a neoantigen candidate was deprioritized when selecting the
Best Peptide. The Best Peptide is highlighted in green.</p>
<p><img src="resources/images/05-pvacview_tour_files/figure-html/1uz39zaObDGKhEVCGzO0JO35CTbC0oRAM0mxgLcMAA9Y_g25ad9ce8c9b_0_32.png" title="The Transcript Set Detailed Data panel shows binding prediction details for the neoantigens the transcripts in the set code for." alt="The Transcript Set Detailed Data panel shows binding prediction details for the neoantigens the transcripts in the set code for." width="100%" style="display: block; margin: auto;" /></p>
<p>The Transcripts in Set tabs shows details of the transcripts in the selected
set such as the transcript Ensembl ID, the transcript expression, the
transcript support level, the biotype, and the transcript length. This
reflects the criteria used in determining the Best Transcript. The Best
Transcript is highlighted in green.</p>
<p><img src="resources/images/05-pvacview_tour_files/figure-html/1uz39zaObDGKhEVCGzO0JO35CTbC0oRAM0mxgLcMAA9Y_g25ad9ce8c9b_0_38.png" title="The Transcripts in Set tab shows details about the transcripts in the set." alt="The Transcripts in Set tab shows details about the transcripts in the set." width="100%" style="display: block; margin: auto;" /></p>
</div>
<div id="additional-peptide-information" class="section level3" number="5.5.4">
<h3><span class="header-section-number">5.5.4</span> Additional Peptide Information</h3>
<p>In the Additional Peptide Information panel, users can see more information
for the neoantigen candidate selected in the Transcript Set Detailed Data
panel.</p>
<p>The first tab, “IC50 Plot”, shows violin plots of the predicted IC50 binding
affinity for each prediction algorithm for both the neoantigen candidate and
its matched wildtype peptide. This can be used to check concordance
of predictions between the different algorithms. It also allows for a
detailed comparison between the mutant and wildtype predictions in addition to
the median or lowest IC50 binding affinity used elsewhere. A solid line is used
to represent the median score.</p>
<p><img src="resources/images/05-pvacview_tour_files/figure-html/1uz39zaObDGKhEVCGzO0JO35CTbC0oRAM0mxgLcMAA9Y_g25ad9ce8c9b_0_44.png" title="The Additional Peptide Information panel shows more information for the peptide selected in the Transcript Set Detailed Data panel." alt="The Additional Peptide Information panel shows more information for the peptide selected in the Transcript Set Detailed Data panel." width="100%" style="display: block; margin: auto;" /></p>
<p>The %ile Plot tab shows a similar violin plot but for the predicted percentile
scores as opposed to the IC50 binding affinity. A solid line is also used here
to represent the median score.</p>
<p><img src="resources/images/05-pvacview_tour_files/figure-html/1uz39zaObDGKhEVCGzO0JO35CTbC0oRAM0mxgLcMAA9Y_g25ad9ce8c9b_0_50.png" title="The %ile Plot tab shows violin plots of the percentile score predicted by each algorithm." alt="The %ile Plot tab shows violin plots of the percentile score predicted by each algorithm." width="100%" style="display: block; margin: auto;" /></p>
<p>The next tab, “Binding Data”, shows the IC50 binding affinity and
percentile score but in table format.</p>
<p><img src="resources/images/05-pvacview_tour_files/figure-html/1uz39zaObDGKhEVCGzO0JO35CTbC0oRAM0mxgLcMAA9Y_g25ad9ce8c9b_0_56.png" title="The Binding Data tab shows a table of the IC50 binding affinity and percentile predicted by each algorithm." alt="The Binding Data tab shows a table of the IC50 binding affinity and percentile predicted by each algorithm." width="100%" style="display: block; margin: auto;" /></p>
<p>The Elution Table tab shows the predicted elution scores and percentiles, if
the appropriate prediction algorithm(s) were chosen.</p>
<p><img src="resources/images/05-pvacview_tour_files/figure-html/1uz39zaObDGKhEVCGzO0JO35CTbC0oRAM0mxgLcMAA9Y_g25ad9ce8c9b_0_62.png" title="The Elution Table tab shows elution prediction scores and precentiles for the selected peptide." alt="The Elution Table tab shows elution prediction scores and precentiles for the selected peptide." width="100%" style="display: block; margin: auto;" /></p>
<p>Lastly, the Anchor Heatmap tab shows a heatmap overlayed over each neoantigen
candidate from the selected transcript set. A darker color represents a higher
probability that a position in the peptide is an anchor. Mutated positions are
represented by red letters. More information
about how to interpret the heatmap can be found in the graph on the right of this
panel.</p>
<p><img src="resources/images/05-pvacview_tour_files/figure-html/1uz39zaObDGKhEVCGzO0JO35CTbC0oRAM0mxgLcMAA9Y_g25ad9ce8c9b_0_68.png" title="The Anchor heatmaps show which positions in a peptide are likely to be anchors." alt="The Anchor heatmaps show which positions in a peptide are likely to be anchors." width="100%" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="regenerate-tiers-with-custom-parameters" class="section level2" number="5.6">
<h2><span class="header-section-number">5.6</span> Regenerate Tiers with Custom Parameters</h2>
<p>During review of your data it might become apparent that different tiering
thresholds would’ve been more approriate. pVACview allows you to re-tier your
data with custom parameters by adjusting the sliders and inputs in the
“Advanced Options: Regenerate Tiering with different parameters” panel and
pressing the “Recalculate Tiering with new parameters” button.</p>
<p>The parameters that were used in the original pVACseq run can still be viewed
in the “Original Parameters for Tiering” panel and the tiers can be reset to
those parameters by pressing the “Reset to original parameters” button.</p>
<p><img src="resources/images/05-pvacview_tour_files/figure-html/1uz39zaObDGKhEVCGzO0JO35CTbC0oRAM0mxgLcMAA9Y_g25ad9ce8c9b_0_87.png" title="Users can re-tier the neoantigen candidates by adjusting the tiering thresholds." alt="Users can re-tier the neoantigen candidates by adjusting the tiering thresholds." width="100%" style="display: block; margin: auto;" /></p>
</div>
<div id="adding-comments-to-variants" class="section level2" number="5.7">
<h2><span class="header-section-number">5.7</span> Adding Comments to Variants</h2>
<p>When reviewing neoantigen candidates in pVACseq, users are able to add
comments on each variant, for example, describing what additional review is
necessary. Please note that comments are not saved until the “Update Comment
Section” button is pressed.</p>
<p><img src="resources/images/05-pvacview_tour_files/figure-html/1uz39zaObDGKhEVCGzO0JO35CTbC0oRAM0mxgLcMAA9Y_g25ad9ce8c9b_0_93.png" title="Users can leave comments on each variant." alt="Users can leave comments on each variant." width="100%" style="display: block; margin: auto;" /></p>
</div>
<div id="exporting-the-aggregated-table" class="section level2" number="5.8">
<h2><span class="header-section-number">5.8</span> Exporting the Aggregated Table</h2>
<p>Once review of the neoantigen candidates has been completed, the results can
be exported by switching to the Export interface via the sidebar. This will
export the Aggregated Report with the updated Evaluation column and comments
added. The report can be exported in either TSV or Excel format.</p>
<p><img src="resources/images/05-pvacview_tour_files/figure-html/1uz39zaObDGKhEVCGzO0JO35CTbC0oRAM0mxgLcMAA9Y_g25ad9ce8c9b_0_99.png" title="Users can leave comments on each variant." alt="Users can leave comments on each variant." width="100%" style="display: block; margin: auto;" /></p>

</div>
</div>
<hr>
<center> 
  <div class="footer">
      All illustrations <a href="https://creativecommons.org/licenses/by/4.0/">CC-BY. </a>
      <br>
      All other materials <a href= "https://creativecommons.org/licenses/by/4.0/"> CC-BY </a> unless noted otherwise.
  </div>
</center>
            </section>

          </div>
        </div>
      </div>
<a href="understanding-pvactools-outputs.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="conclusions.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
