
# Tour of pVACview

```{r, include = FALSE}
ottrpal::set_knitr_image_path()
```

## Learning Objectives

This chapter will cover:  

- Introduction to the pVACview module
- Demo of the pVACview interface

## Introduction to the pVACview module

pVACview is a R shiny based tool designed to aid specifically in the prioritization and selection of neoantigen candidates for personalized cancer vaccines. It takes as inputs a pVACseq output aggregate report file (tsv format) and a corresponding pVACseq output metrics file (json). pVACview allows the user to launch an R shiny application to load and visualize the given neoantigen candidates with detailed information including that of the genomic variant, transcripts covering the variant, and good-binding peptides predicted from the respective transcripts. It also incorporates anchor prediction data for a range of class I HLA alleles and peptides ranging from 8 to 11-mers. By taking all levels of information into account for the neoantigen candidates, clinicians will be able to make more informed decisions when deciding final peptide candidates for personalized cancer vaccines.

```{r, fig.align='center', out.width="100%", fig.width="100%", echo = FALSE, fig.alt= "Upon successfully uploading the relevant data files, you can explore the different aspects of your neoantigen candidates."}
ottrpal::include_slide("https://docs.google.com/presentation/d/1uz39zaObDGKhEVCGzO0JO35CTbC0oRAM0mxgLcMAA9Y/edit#slide=id.g2491f283519_0_8")
```

## Tour of the pVACview interface

Here is a brief tour of the [pVACview](https://pvactools.readthedocs.io/en/latest/pvacview.html){target="_blank"} interface:
 
```{r, echo=FALSE}
knitr::include_url("https://www.youtube.com/embed/SMcXSV1cp1U")
```

