<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 1 Introduction | Introduction to pVACtools</title>
  <meta name="description" content="An introduction to the background and basic concepts around neoantigen identification and how to use pVACtools" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 1 Introduction | Introduction to pVACtools" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="An introduction to the background and basic concepts around neoantigen identification and how to use pVACtools" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 1 Introduction | Introduction to pVACtools" />
  
  <meta name="twitter:description" content="An introduction to the background and basic concepts around neoantigen identification and how to use pVACtools" />
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="assets/ITN_favicon.ico" type="image/x-icon" />
<link rel="prev" href="index.html"/>
<link rel="next" href="prerequisites.html"/>
<script src="libs/header-attrs-2.10/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  background-color: #f8f8f8; }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #c4a000; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #000000; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #000000; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #000000; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="assets/style_ITN.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<a href="https://www.itcrtraining.org/"><img src="assets/ITN_logo.png" style="padding-left: 15px; padding-top: 8px;"</a>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>About this Course</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#motivation"><i class="fa fa-check"></i><b>1.1</b> Motivation</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#background"><i class="fa fa-check"></i><b>1.2</b> Background</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#target-audience"><i class="fa fa-check"></i><b>1.3</b> Target Audience</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#curriculum"><i class="fa fa-check"></i><b>1.4</b> Curriculum</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="prerequisites.html"><a href="prerequisites.html"><i class="fa fa-check"></i><b>2</b> Prerequisites</a>
<ul>
<li class="chapter" data-level="2.1" data-path="prerequisites.html"><a href="prerequisites.html#learning-objectives"><i class="fa fa-check"></i><b>2.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="2.2" data-path="prerequisites.html"><a href="prerequisites.html#docker"><i class="fa fa-check"></i><b>2.2</b> Docker</a></li>
<li class="chapter" data-level="2.3" data-path="prerequisites.html"><a href="prerequisites.html#terminal"><i class="fa fa-check"></i><b>2.3</b> Terminal</a></li>
<li class="chapter" data-level="2.4" data-path="prerequisites.html"><a href="prerequisites.html#r-studio-and-r-package-dependencies"><i class="fa fa-check"></i><b>2.4</b> R Studio and R package dependencies</a></li>
<li class="chapter" data-level="2.5" data-path="prerequisites.html"><a href="prerequisites.html#data"><i class="fa fa-check"></i><b>2.5</b> Data</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="running-pvactools.html"><a href="running-pvactools.html"><i class="fa fa-check"></i><b>3</b> Running pVACtools</a>
<ul>
<li class="chapter" data-level="3.1" data-path="running-pvactools.html"><a href="running-pvactools.html#learning-objectives-1"><i class="fa fa-check"></i><b>3.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="3.2" data-path="running-pvactools.html"><a href="running-pvactools.html#starting-docker"><i class="fa fa-check"></i><b>3.2</b> Starting Docker</a></li>
<li class="chapter" data-level="3.3" data-path="running-pvactools.html"><a href="running-pvactools.html#running-pvacseq"><i class="fa fa-check"></i><b>3.3</b> Running pVACseq</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="running-pvactools.html"><a href="running-pvactools.html#required-parameters-for-pvacseq"><i class="fa fa-check"></i><b>3.3.1</b> Required Parameters for pVACseq</a></li>
<li class="chapter" data-level="3.3.2" data-path="running-pvactools.html"><a href="running-pvactools.html#optional-parameters-for-pvacseq"><i class="fa fa-check"></i><b>3.3.2</b> Optional Parameters for pVACseq</a></li>
<li class="chapter" data-level="3.3.3" data-path="running-pvactools.html"><a href="running-pvactools.html#pvacseq-command"><i class="fa fa-check"></i><b>3.3.3</b> pVACseq Command</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="running-pvactools.html"><a href="running-pvactools.html#running-pvacfuse"><i class="fa fa-check"></i><b>3.4</b> Running pVACfuse</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="running-pvactools.html"><a href="running-pvactools.html#required-parameters-for-pvacfuse"><i class="fa fa-check"></i><b>3.4.1</b> Required Parameters for pVACfuse</a></li>
<li class="chapter" data-level="3.4.2" data-path="running-pvactools.html"><a href="running-pvactools.html#optional-parameters-for-pvacfuse"><i class="fa fa-check"></i><b>3.4.2</b> Optional Parameters for pVACfuse</a></li>
<li class="chapter" data-level="3.4.3" data-path="running-pvactools.html"><a href="running-pvactools.html#pvacfuse-command"><i class="fa fa-check"></i><b>3.4.3</b> pVACfuse Command</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html"><i class="fa fa-check"></i><b>4</b> Understanding pVACtools outputs</a>
<ul>
<li class="chapter" data-level="4.1" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#learning-objectives-2"><i class="fa fa-check"></i><b>4.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="4.2" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#pvactools-output-files"><i class="fa fa-check"></i><b>4.2</b> pVACtools Output Files</a></li>
<li class="chapter" data-level="4.3" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#interpreting-the-filtered.tsv-file"><i class="fa fa-check"></i><b>4.3</b> Interpreting the filtered.tsv File</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#binding-filter"><i class="fa fa-check"></i><b>4.3.1</b> Binding Filter</a></li>
<li class="chapter" data-level="4.3.2" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#coverage-filter"><i class="fa fa-check"></i><b>4.3.2</b> Coverage Filter</a></li>
<li class="chapter" data-level="4.3.3" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#transcript-support-level-filter"><i class="fa fa-check"></i><b>4.3.3</b> Transcript Support Level Filter</a></li>
<li class="chapter" data-level="4.3.4" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#top-score-filter"><i class="fa fa-check"></i><b>4.3.4</b> Top Score Filter</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#interpreting-the-aggregated.tsv-file"><i class="fa fa-check"></i><b>4.4</b> Interpreting the aggregated.tsv File</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#determining-the-best-transcript-and-best-peptide-of-a-variant"><i class="fa fa-check"></i><b>4.4.1</b> Determining the Best Transcript and Best Peptide of a Variant</a></li>
<li class="chapter" data-level="4.4.2" data-path="understanding-pvactools-outputs.html"><a href="understanding-pvactools-outputs.html#tier-and-tiering-criteria"><i class="fa fa-check"></i><b>4.4.2</b> Tier and Tiering Criteria</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="tour-of-pvacview.html"><a href="tour-of-pvacview.html"><i class="fa fa-check"></i><b>5</b> Tour of pVACview</a>
<ul>
<li class="chapter" data-level="5.1" data-path="tour-of-pvacview.html"><a href="tour-of-pvacview.html#learning-objectives-3"><i class="fa fa-check"></i><b>5.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="5.2" data-path="tour-of-pvacview.html"><a href="tour-of-pvacview.html#introduction-to-the-pvacview-module"><i class="fa fa-check"></i><b>5.2</b> Introduction to the pVACview module</a></li>
<li class="chapter" data-level="5.3" data-path="tour-of-pvacview.html"><a href="tour-of-pvacview.html#tour-of-the-pvacview-interface"><i class="fa fa-check"></i><b>5.3</b> Tour of the pVACview interface</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>6</b> Conclusions</a>
<ul>
<li class="chapter" data-level="6.1" data-path="conclusions.html"><a href="conclusions.html#learning-objectives-4"><i class="fa fa-check"></i><b>6.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="6.2" data-path="conclusions.html"><a href="conclusions.html#key-conclusions"><i class="fa fa-check"></i><b>6.2</b> Key conclusions</a></li>
<li class="chapter" data-level="6.3" data-path="conclusions.html"><a href="conclusions.html#additional-resources"><i class="fa fa-check"></i><b>6.3</b> Additional Resources</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="about-the-authors.html"><a href="about-the-authors.html"><i class="fa fa-check"></i>About the Authors</a></li>
<li class="chapter" data-level="7" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>7</b> References</a></li>
<li class="divider"></li>
<p style="text-align:center;"> <a href="https://github.com/jhudsl/OTTR_Template" target="blank" > This content was published with</a> <a href="https://bookdown.org/" target="blank"> bookdown by: </a> </p>
<p style="text-align:center;"> <a href="https://griffithlab.org/"> The Griffith Lab </a></p>
<p style="text-align:center; font-size: 12px;"> <a href="https://github.com/rstudio4edu/rstudio4edu-book/"> Style adapted from: rstudio4edu-book </a> <a href ="https://creativecommons.org/licenses/by/2.0/"> (CC-BY 2.0) </a></p>
<p style="padding-left: 40px;"><div class="trapezoid" style = "padding-left: 40px;"><span>  <a href="https://forms.gle/hc8Xt3Y2Znjb6M4Y7"> Click here to provide feedback</a> <img src="assets/itcr_arrow.png" style=" width: 10%" ></span></div></p>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Introduction to pVACtools</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<head>
  <meta name="viewport" content="width=device-width,minimum-scale=1.0,maximum-scale=1.0,initial-scale=1.0">
  <!--script src="https://kit.fontawesome.com/6a26f47516.js"></script-->
  <!--<script src="assets/hideOutput.js"></script>-->
  <link href="assets/style_ITN.css" rel="stylesheet">
</head>
        


<div class="hero-image-container"> 
  <img class= "hero-image" src="assets/itcr_main_image.png">
</div>
<div id="introduction" class="section level1" number="1">
<h1><span class="header-section-number">Chapter 1</span> Introduction</h1>
<p>This course has been developed recently (Summer 2023). We welcome any feedback at <a href="mailto:help@pvactools.org" class="email">help@pvactools.org</a> or by submission of <a href="https://github.com/griffithlab/pVACtools_Intro_Course/issues">GitHub issues</a>.</p>
<div id="motivation" class="section level2" number="1.1">
<h2><span class="header-section-number">1.1</span> Motivation</h2>
<p>Identification of neoantigens is a critical step in predicting response to checkpoint blockade therapy and design of personalized cancer vaccines.
This is a cross-disciplinary challenge, involving genomics, proteomics, immunology, and computational approaches. We have built a computational
framework called pVACtools that, when paired with a well-established genomics pipeline, produces an end-to-end solution for neoantigen characterization.
pVACtools supports identification of altered peptides from different mechanisms, including point mutations, in-frame and frameshift insertions and deletions,
and gene fusions. Prediction of peptide:MHC binding is accomplished by supporting an ensemble of MHC Class I and II binding algorithms within a framework
designed to facilitate the incorporation of additional algorithms. Prioritization of predicted peptides occurs by integrating diverse data, including mutant
allele expression, peptide binding affinities, and determination of whether a mutation is clonal or subclonal. Interactive visualization via a Web interface allows
users to efficiently generate, review, and interpret results, selecting candidate peptides for individual experiments or patient vaccine designs. Additional modules
support design choices needed for competing vaccine delivery approaches. One such module optimizes peptide ordering to minimize junctional epitopes in DNA vector
vaccines. Downstream analysis commands for synthetic long peptide vaccines are available to assess candidates for factors that influence peptide synthesis. All
of the aforementioned steps are executed via a modular workflow consisting of tools for neoantigen prediction from somatic alterations (pVACseq, pVACfuse, and pVACbind),
prioritization, and selection using a graphical Web-based interface (pVACview), and design of DNA vector–based vaccines (pVACvector) and synthetic long peptide
vaccines. pVACtools is available at <a href="http://www.pvactools.org">http://www.pvactools.org</a>.</p>
<p><img src="resources/images/01-intro_files/figure-html/1uz39zaObDGKhEVCGzO0JO35CTbC0oRAM0mxgLcMAA9Y_g22b1533a196_0_0.png" title="pVACtools is a cancer immunotherapy tools suite" alt="pVACtools is a cancer immunotherapy tools suite" width="100%" style="display: block; margin: auto;" /></p>
</div>
<div id="background" class="section level2" number="1.2">
<h2><span class="header-section-number">1.2</span> Background</h2>
<p>Neoantigens are unique peptide sequences generated from mutations acquired somatically in tumor cells. These antigens provide an avenue for tumor-specific immune
cell recognition and have been found to be important targets for cancer immunotherapies <span class="citation">(<a href="#ref-Keskin2018" role="doc-biblioref">Keskin et al. 2018</a>; <a href="#ref-Ott2017" role="doc-biblioref">Ott et al. 2017</a>; <a href="#ref-Hilf2018" role="doc-biblioref">Hilf et al. 2018</a>)</span>. Effective neoantigens, presented by the
major histocompatibility complex (MHC) and thus introduced to the patient’s immune system, can prime and activate CD8+ and CD4+ T cells for downstream signaling of
cell-death. Patients with high tumor mutation burden tend to have stronger responses to neoantigen based immunotherapy treatments <span class="citation">(<a href="#ref-Brown2014" role="doc-biblioref">Brown et al. 2014</a>; <a href="#ref-Rizvi2015" role="doc-biblioref">Rizvi et al. 2015</a>; <a href="#ref-Schumacher2015" role="doc-biblioref">Schumacher and Schreiber 2015</a>)</span>.
DNA and RNA sequencing technologies allow researchers and clinicians to computationally predict potential neoantigens based on tumor-specific mutations.</p>
<p>However, neoantigen generation and presentation is complex, and a host of factors must be evaluated by complex analyses to characterize each potential neoantigen.
These include but are not limited to: somatic variant identification, tumor clonality assessment, RNA expression estimation, mRNA isoform selection, inference of
translated tumor specific peptides that arise from the somatic variant, and prediction of peptide processing, peptide transportation, peptide-MHC binding, peptide-MHC
stability and recognition by cytotoxic T cells <span class="citation">(<a href="#ref-Richters2019" role="doc-biblioref">Richters et al. 2019</a>)</span>.</p>
<p>pVACtools can be used as the final step in a well-established variant calling pipeline. It leverages existing tools with functionality related to variant annotation
(Ensembl VEP <span class="citation">(<a href="#ref-McLaren2016" role="doc-biblioref">McLaren et al. 2016</a>)</span>), identifying neoantigens from specific sources (e.g. fusions via star-fusion <span class="citation">(<a href="#ref-Haas2019" role="doc-biblioref">Haas et al. 2019</a>)</span>, AGFusion <span class="citation">(<a href="#ref-Murphy2016" role="doc-biblioref">Murphy and Elemento 2016</a>)</span>, and Arriba <span class="citation">(<a href="#ref-Uhrig2021" role="doc-biblioref">Uhrig et al. 2021</a>)</span>),
HLA typing (OptiType <span class="citation">(<a href="#ref-Szolek2014" role="doc-biblioref">Szolek et al. 2014</a>)</span>, PHLAT <span class="citation">(<a href="#ref-Bai2018" role="doc-biblioref">Bai, Wang, and Fury 2018</a>)</span>), peptide-MHC binding prediction (IEDB <span class="citation">(<a href="#ref-Vita2018" role="doc-biblioref">Vita et al. 2018</a>)</span>, NetMHCpan <span class="citation">(<a href="#ref-Jurtz2017" role="doc-biblioref">Jurtz et al. 2017</a>)</span>, MHCflurry <span class="citation">(<a href="#ref-ODonnell2018" role="doc-biblioref">ODonnell et al. 2018</a>)</span>,
MHCnuggets <span class="citation">(<a href="#ref-Shao2020" role="doc-biblioref">Shao et al. 2020</a>)</span>), peptide-MHC stability (NetMHCstabpan <span class="citation">(<a href="#ref-Rasmussen2016" role="doc-biblioref">Rasmussen et al. 2016</a>)</span>], peptide processing (NetChop <span class="citation">(<a href="#ref-Nielsen2005" role="doc-biblioref">Nielsen et al. 2005</a>)</span>), manufacturability
metrics (vaxrank <span class="citation">(<a href="#ref-Rubinsteyn2017" role="doc-biblioref">Rubinsteyn et al. 2017</a>)</span>), and reference proteome similarity (BLAST <span class="citation">(<a href="#ref-Altschul1990" role="doc-biblioref">Altschul et al. 1990</a>)</span>). Each of these tools tackles specific tasks within the broader goal of
antigen analysis and is utilized by pVACtools to provide an end-to-end integration of novel algorithms and established tools needed to discover, characterize, prioritize,
and utilize tumor-specific neoantigens in basic research and clinical applications. Combining pVACtools with existing variant calling pipelines provides an end-to-end
solution for neoantigen prediction and characterization.</p>
<p><img src="resources/images/01-intro_files/figure-html/1uz39zaObDGKhEVCGzO0JO35CTbC0oRAM0mxgLcMAA9Y_g22b1533a196_0_6.png" title="Tumor neoantigen background" alt="Tumor neoantigen background" width="100%" style="display: block; margin: auto;" /></p>
</div>
<div id="target-audience" class="section level2" number="1.3">
<h2><span class="header-section-number">1.3</span> Target Audience</h2>
<p>The course is intended for anyone seeking a better understanding of current best practices in neoantigen identification and prioritization using pVACtools.
It assumes that the learner is familiar with basic biology, genetics and immunology concepts.</p>
</div>
<div id="curriculum" class="section level2" number="1.4">
<h2><span class="header-section-number">1.4</span> Curriculum</h2>
<p>This course will teach learners to:</p>
<ul>
<li>Understand key concepts of immunogenomics and neoantigen identification</li>
<li>How to run the pVACtools software suite</li>
<li>How to visualize and prioritize neoantigen candidates with pVACview</li>
</ul>
<p>For more background on pVACtools please consult the core pVAC papers <span class="citation">(<a href="#ref-Hundal2016" role="doc-biblioref">Hundal et al. 2016</a>, <a href="#ref-Hundal2018" role="doc-biblioref">2018</a>, <a href="#ref-Hundal2020" role="doc-biblioref">2020</a>)</span>.</p>

</div>
</div>
<h3> References</h3>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-Altschul1990" class="csl-entry">
Altschul, Stephen F., Warren Gish, Webb Miller, Eugene W. Myers, and David J. Lipman. 1990. <span>“Basic Local Alignment Search Tool.”</span> <em>Journal of Molecular Biology</em> 215 (3): 403–10. <a href="https://doi.org/10.1016/s0022-2836(05)80360-2">https://doi.org/10.1016/s0022-2836(05)80360-2</a>.
</div>
<div id="ref-Bai2018" class="csl-entry">
Bai, Yu, David Wang, and Wen Fury. 2018. <span>“<span>PHLAT</span>: Inference of High-Resolution <span>HLA</span> Types from <span>RNA</span> and Whole Exome Sequencing.”</span> In <em>Methods in Molecular Biology</em>, 193–201. Springer New York. <a href="https://doi.org/10.1007/978-1-4939-8546-3_13">https://doi.org/10.1007/978-1-4939-8546-3_13</a>.
</div>
<div id="ref-Brown2014" class="csl-entry">
Brown, Scott D., Rene L. Warren, Ewan A. Gibb, Spencer D. Martin, John J. Spinelli, Brad H. Nelson, and Robert A. Holt. 2014. <span>“Neo-Antigens Predicted by Tumor Genome Meta-Analysis Correlate with Increased Patient Survival.”</span> <em>Genome Research</em> 24 (5): 743–50. <a href="https://doi.org/10.1101/gr.165985.113">https://doi.org/10.1101/gr.165985.113</a>.
</div>
<div id="ref-Haas2019" class="csl-entry">
Haas, Brian J., Alexander Dobin, Bo Li, Nicolas Stransky, Nathalie Pochet, and Aviv Regev. 2019. <span>“Accuracy Assessment of Fusion Transcript Detection via Read-Mapping and de Novo Fusion Transcript Assembly-Based Methods.”</span> <em>Genome Biology</em> 20 (1). <a href="https://doi.org/10.1186/s13059-019-1842-9">https://doi.org/10.1186/s13059-019-1842-9</a>.
</div>
<div id="ref-Hilf2018" class="csl-entry">
Hilf, Norbert, Sabrina Kuttruff-Coqui, Katrin Frenzel, Valesca Bukur, Stefan Stevanović, Cécile Gouttefangeas, Michael Platten, et al. 2018. <span>“Actively Personalized Vaccination Trial for Newly Diagnosed Glioblastoma.”</span> <em>Nature</em> 565 (7738): 240–45. <a href="https://doi.org/10.1038/s41586-018-0810-y">https://doi.org/10.1038/s41586-018-0810-y</a>.
</div>
<div id="ref-Hundal2016" class="csl-entry">
Hundal, Jasreet, Beatriz M. Carreno, Allegra A. Petti, Gerald P. Linette, Obi L. Griffith, Elaine R. Mardis, and Malachi Griffith. 2016. <span>“<span class="nocase">pVAC</span>-Seq: A Genome-Guided in Silico Approach to Identifying Tumor Neoantigens.”</span> <em>Genome Medicine</em> 8 (1). <a href="https://doi.org/10.1186/s13073-016-0264-5">https://doi.org/10.1186/s13073-016-0264-5</a>.
</div>
<div id="ref-Hundal2018" class="csl-entry">
Hundal, Jasreet, Susanna Kiwala, Yang-Yang Feng, Connor J. Liu, Ramaswamy Govindan, William C. Chapman, Ravindra Uppaluri, et al. 2018. <span>“Accounting for Proximal Variants Improves Neoantigen Prediction.”</span> <em>Nature Genetics</em> 51 (1): 175–79. <a href="https://doi.org/10.1038/s41588-018-0283-9">https://doi.org/10.1038/s41588-018-0283-9</a>.
</div>
<div id="ref-Hundal2020" class="csl-entry">
Hundal, Jasreet, Susanna Kiwala, Joshua McMichael, Christopher A. Miller, Huiming Xia, Alexander T. Wollam, Connor J. Liu, et al. 2020. <span>“<span class="nocase">pVACtools</span>: A Computational Toolkit to Identify and Visualize Cancer Neoantigens.”</span> <em>Cancer Immunology Research</em> 8 (3): 409–20. <a href="https://doi.org/10.1158/2326-6066.cir-19-0401">https://doi.org/10.1158/2326-6066.cir-19-0401</a>.
</div>
<div id="ref-Jurtz2017" class="csl-entry">
Jurtz, Vanessa, Sinu Paul, Massimo Andreatta, Paolo Marcatili, Bjoern Peters, and Morten Nielsen. 2017. <span>“<span>NetMHCpan</span> 4.0: Improved Peptide-<span>MHC</span> Class i Interaction Predictions Integrating Eluted Ligand and Peptide Binding Affinity Data,”</span> June. <a href="https://doi.org/10.1101/149518">https://doi.org/10.1101/149518</a>.
</div>
<div id="ref-Keskin2018" class="csl-entry">
Keskin, Derin B., Annabelle J. Anandappa, Jing Sun, Itay Tirosh, Nathan D. Mathewson, Shuqiang Li, Giacomo Oliveira, et al. 2018. <span>“Neoantigen Vaccine Generates Intratumoral t Cell Responses in Phase Ib Glioblastoma Trial.”</span> <em>Nature</em> 565 (7738): 234–39. <a href="https://doi.org/10.1038/s41586-018-0792-9">https://doi.org/10.1038/s41586-018-0792-9</a>.
</div>
<div id="ref-McLaren2016" class="csl-entry">
McLaren, William, Laurent Gil, Sarah E. Hunt, Harpreet Singh Riat, Graham R. S. Ritchie, Anja Thormann, Paul Flicek, and Fiona Cunningham. 2016. <span>“The Ensembl Variant Effect Predictor.”</span> <em>Genome Biology</em> 17 (1). <a href="https://doi.org/10.1186/s13059-016-0974-4">https://doi.org/10.1186/s13059-016-0974-4</a>.
</div>
<div id="ref-Murphy2016" class="csl-entry">
Murphy, Charlie, and Olivier Elemento. 2016. <span>“<span>AGFusion</span>: Annotate and Visualize Gene Fusions,”</span> October. <a href="https://doi.org/10.1101/080903">https://doi.org/10.1101/080903</a>.
</div>
<div id="ref-Nielsen2005" class="csl-entry">
Nielsen, Morten, Claus Lundegaard, Ole Lund, and Can Keşmir. 2005. <span>“The Role of the Proteasome in Generating Cytotoxic t-Cell Epitopes: Insights Obtained from Improved Predictions of Proteasomal Cleavage.”</span> <em>Immunogenetics</em> 57 (1-2): 33–41. <a href="https://doi.org/10.1007/s00251-005-0781-7">https://doi.org/10.1007/s00251-005-0781-7</a>.
</div>
<div id="ref-ODonnell2018" class="csl-entry">
ODonnell, Timothy J., Alex Rubinsteyn, Maria Bonsack, Angelika B. Riemer, Uri Laserson, and Jeff Hammerbacher. 2018. <span>“<span>MHCflurry</span>: Open-Source Class i <span>MHC</span> Binding Affinity Prediction.”</span> <em>Cell Systems</em> 7 (1): 129–132.e4. <a href="https://doi.org/10.1016/j.cels.2018.05.014">https://doi.org/10.1016/j.cels.2018.05.014</a>.
</div>
<div id="ref-Ott2017" class="csl-entry">
Ott, Patrick A., Zhuting Hu, Derin B. Keskin, Sachet A. Shukla, Jing Sun, David J. Bozym, Wandi Zhang, et al. 2017. <span>“An Immunogenic Personal Neoantigen Vaccine for Patients with Melanoma.”</span> <em>Nature</em> 547 (7662): 217–21. <a href="https://doi.org/10.1038/nature22991">https://doi.org/10.1038/nature22991</a>.
</div>
<div id="ref-Rasmussen2016" class="csl-entry">
Rasmussen, Michael, Emilio Fenoy, Mikkel Harndahl, Anne Bregnballe Kristensen, Ida Kallehauge Nielsen, Morten Nielsen, and Søren Buus. 2016. <span>“Pan-Specific Prediction of Peptide<span></span><span>MHC</span> Class i Complex Stability, a Correlate of t Cell Immunogenicity.”</span> <em>The Journal of Immunology</em> 197 (4): 1517–24. <a href="https://doi.org/10.4049/jimmunol.1600582">https://doi.org/10.4049/jimmunol.1600582</a>.
</div>
<div id="ref-Richters2019" class="csl-entry">
Richters, Megan M., Huiming Xia, Katie M. Campbell, William E. Gillanders, Obi L. Griffith, and Malachi Griffith. 2019. <span>“Best Practices for Bioinformatic Characterization of Neoantigens for Clinical Utility.”</span> <em>Genome Medicine</em> 11 (1). <a href="https://doi.org/10.1186/s13073-019-0666-2">https://doi.org/10.1186/s13073-019-0666-2</a>.
</div>
<div id="ref-Rizvi2015" class="csl-entry">
Rizvi, Naiyer A., Matthew D. Hellmann, Alexandra Snyder, Pia Kvistborg, Vladimir Makarov, Jonathan J. Havel, William Lee, et al. 2015. <span>“Mutational Landscape Determines Sensitivity to <span>PD</span>-1 Blockade in Non<span></span>small Cell Lung Cancer.”</span> <em>Science</em> 348 (6230): 124–28. <a href="https://doi.org/10.1126/science.aaa1348">https://doi.org/10.1126/science.aaa1348</a>.
</div>
<div id="ref-Rubinsteyn2017" class="csl-entry">
Rubinsteyn, Alexander, Isaac Hodes, Julia Kodysh, and Jeffrey Hammerbacher. 2017. <span>“Vaxrank: A Computational Tool for Designing Personalized Cancer Vaccines,”</span> May. <a href="https://doi.org/10.1101/142919">https://doi.org/10.1101/142919</a>.
</div>
<div id="ref-Schumacher2015" class="csl-entry">
Schumacher, Ton N., and Robert D. Schreiber. 2015. <span>“Neoantigens in Cancer Immunotherapy.”</span> <em>Science</em> 348 (6230): 69–74. <a href="https://doi.org/10.1126/science.aaa4971">https://doi.org/10.1126/science.aaa4971</a>.
</div>
<div id="ref-Shao2020" class="csl-entry">
Shao, Xiaoshan M., Rohit Bhattacharya, Justin Huang, I. K. Ashok Sivakumar, Collin Tokheim, Lily Zheng, Dylan Hirsch, et al. 2020. <span>“High-Throughput Prediction of <span>MHC</span> Class i and <span>II</span> Neoantigens with <span>MHCnuggets</span>.”</span> <em>Cancer Immunology Research</em> 8 (3): 396–408. <a href="https://doi.org/10.1158/2326-6066.cir-19-0464">https://doi.org/10.1158/2326-6066.cir-19-0464</a>.
</div>
<div id="ref-Szolek2014" class="csl-entry">
Szolek, András, Benjamin Schubert, Christopher Mohr, Marc Sturm, Magdalena Feldhahn, and Oliver Kohlbacher. 2014. <span>“<span>OptiType</span>: Precision <span>HLA</span> Typing from Next-Generation Sequencing Data.”</span> <em>Bioinformatics</em> 30 (23): 3310–16. <a href="https://doi.org/10.1093/bioinformatics/btu548">https://doi.org/10.1093/bioinformatics/btu548</a>.
</div>
<div id="ref-Uhrig2021" class="csl-entry">
Uhrig, Sebastian, Julia Ellermann, Tatjana Walther, Pauline Burkhardt, Martina Fröhlich, Barbara Hutter, Umut H. Toprak, et al. 2021. <span>“Accurate and Efficient Detection of Gene Fusions from <span>RNA</span> Sequencing Data.”</span> <em>Genome Research</em> 31 (3): 448–60. <a href="https://doi.org/10.1101/gr.257246.119">https://doi.org/10.1101/gr.257246.119</a>.
</div>
<div id="ref-Vita2018" class="csl-entry">
Vita, Randi, Swapnil Mahajan, James A Overton, Sandeep Kumar Dhanda, Sheridan Martini, Jason R Cantrell, Daniel K Wheeler, Alessandro Sette, and Bjoern Peters. 2018. <span>“The Immune Epitope Database (<span>IEDB</span>): 2018 Update.”</span> <em>Nucleic Acids Research</em> 47 (D1): D339–43. <a href="https://doi.org/10.1093/nar/gky1006">https://doi.org/10.1093/nar/gky1006</a>.
</div>
</div>
<hr>
<center> 
  <div class="footer">
      All illustrations <a href="https://creativecommons.org/licenses/by/4.0/">CC-BY. </a>
      <br>
      All other materials <a href= "https://creativecommons.org/licenses/by/4.0/"> CC-BY </a> unless noted otherwise.
  </div>
</center>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="prerequisites.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
